packages/keystone/lib/Keystone/index.js

```diff
@@ -57,6 +57,11 @@ module.exports = class Keystone {
     secureCookies = process.env.NODE_ENV === 'production', // Default to true in production
     cookieMaxAge = 1000 * 60 * 60 * 24 * 30, // 30 days
     schemaNames = ['public'],
+    appVersion = {
+      version: '1.0.0',
+      addVersionToHttpHeaders: true,
```

Comment:
I'd love this variable so somehow clarify that the info is added to the headers of _all_, regardless of the access controls configured. Something like `addVersionToHttpHeadersForAllRequests` but less wordy.. :/

Reply:
The docs currently say "If `addVersionToHttpHeaders` is `true` then all requests will have the header `X-Keystone-App-Version` set.". Is there some way we can make that any more clear?